canvas_modules/harness/cypress/support/properties/properties-verification-cmds.js

```diff
@@ -60,43 +60,73 @@ Cypress.Commands.add("verifyPropertiesFlyoutDoesNotExist", () => {
 });
 
 /** Verify the tooltip over the given text is 'visible'
-* @param label: label of the container shown in the UI
+* @param container: the container shown in the UI
+* @param visible: "visible" or "hidden"
 * @param text: text shown in the tip
+* @param tipLocation: the location of the tooltip
 */
-Cypress.Commands.add("verifyTipForLabelIsVisibleAtLocation", (label, tipLocation, text) => {
-	// Verify the tip for label "Mode" is visible on the "top" with text "Include or discard rows"
-	cy.getControlContainerFromName(label)
```

Comment:
Most of this was duplicate so I created 1 method to handle tooltip verification.